packages/shared/src/styles/components/buttons.css

```diff
@@ -31,6 +31,14 @@
     border-radius: 0.875rem;
   }
 
+  &.headline {
+    height: 3rem;
+    padding: 0 1.5rem;
+    font-size: 1.0625rem;
+    border-radius: 0.875rem;
+    color: var(--theme-label-primary);
+  }
```

Comment:
Is this a recurring thing?
Why don't we set them as classNames?

Reply:
It is due to using `primary`, `secondary`, etc., automatically applying their own color of the label, this is to force it, regardless of which type of button was used.